kcc_cli/keyboard_backlight.py

```diff
@@ -0,0 +1,89 @@
+import subprocess
+from time import sleep
+from typing import Dict
+
+from kcc_cli.common import read_file, write_file
+from kcc_cli.enums import Position, Mode
+
+
+def get_laptop_model() -> str:
+    return subprocess.check_output(['sudo', 'dmidecode', '-s', 'system-product-name']).decode('utf-8').strip()
+
+
+class KeyboardBacklight:
+    MODEL_BACKLIGHT_PATH_MAPPING = {
+        'Oryx Pro': {
+            'brightness_path': '/sys/class/leds/system76_acpi::kbd_backlight/brightness',
```

Comment:
For the `oryp4` this would be the same path as the Serval WS that we have below.